src/Verilator.cpp

```diff
@@ -133,8 +133,6 @@ static void process() {
     v3Global.opt.checkParameters();
     // Correct state we couldn't know at parse time, repair SEL's
     V3LinkResolve::linkResolve(v3Global.rootp());
-    // Set Lvalue's in variable refs
-    V3LinkLValue::linkLValue(v3Global.rootp());
```

Comment:
V3Simulate, used by parameterification seems to care about the settings, so I think this is still needed.

Reply:
I suppose that you are talking about using the simulation node under V3Const, which is used to constify parameters, right?

From what I've seen, the only place so far in which lvalueness it seems to matter is some code in visitor override for `AstVarRef` in `SimulateVisitor`. I wonder if this is something that can be worked around.

I'll revert that change for now as I don't know if it's safe to skip this stage.

Do you have any ideas what kind of scenario could break it, given that all tests pass?